AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -181,6 +183,7 @@ - (void)removePlayerItemObservers:(AVPlayerItem *)playerItem
 
   NSNotificationCenter *notificationCenter = [NSNotificationCenter defaultCenter];
   [notificationCenter removeObserver:self name:AVPlayerItemDidPlayToEndTimeNotification object:playerItem];
+  [notificationCenter removeObserver:self name: AVPlayerItemPlaybackStalledNotification object:playerItem];
```

Comment:
Very tiny typo: instead of space before AVPlayer, you could add an extra space before object, if the goal is to line up the colons (optional)